### DISCUSSION TITLE
postgres-operator as highly available

### DISCUSSION THREAD

**Discussion by uk1988** [zalando/postgres-operator#2184] [Q&A]
We are running postgres-operator in our small k8s clusters which are installed on-prem.
The cluster is set up as a 3 nodes cluster with all nodes running all k8s roles.
Per definition, postgres-operator has a single replica count. Is it possible to run postgres-operator replica count higher as 1? 

With postgres-operator we deploy 2 DB one for keycloak and one for our application. 
Both of the DB we do run as HA.  However,  postgres-operator we run only as a single pod since replica count is not a config parameter of the operator.  What is the impact if the nodes where postgres-operator is running goes down for 2 hours?

**Comment by jfmatth** [comment]
does the operator have any bearing on a running cluster?  I would think not?

**Reply by uk1988** [reply]
Yes in the end that is the question. If there is no bearing on a running cluster then there is no problem if it fails.
